boards/microchip/mec_assy6941/Kconfig.defconfig

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2025 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_MEC_ASSY6941_MEC1743QLJ || BOARD_MEC_ASSY6941_MEC1743QSZ ||\
+	BOARD_MEC_ASSY6941_MEC1753QLJ || BOARD_MEC_ASSY6941_MEC1753QSZ
+
+if RTOS_TIMER
+config SYS_CLOCK_HW_CYCLES_PER_SEC
```

Comment:
newline gap after if

Reply:
Removed.  Kernel timer configs moved to soc layer.

Reply:
the changes need squashing into the correct commit, the original commit still has this and is wrong

Reply:
Squashed mec_assy6941 commits into one. Should be resolved.